d2l-scroll-wrapper.html

```diff
@@ -349,9 +349,18 @@
 					showActions: Boolean
 				},
 
-				listeners: {
-					'iron-resize': 'checkScrollbar',
-					'wrapper.scroll': 'checkScrollThresholds'
+				attached: function() {
+					// Defer the offsetWidth/scrollWidth calculations until after the page has rendered
+					Polymer.RenderStatus.afterNextRender(this, function() {
+						this.listen(this, 'iron-resize', 'checkScrollbar');
+						this.listen(this.$.wrapper, 'scroll', 'checkScrollThresholds');
```

Comment:
Not a big deal, but it seems like you could wire up the event listeners outside of `afterNextRender` and only keep `checkScrollbar()` in here... or does just wiring them up cause a paint? This causes a small period of time where the listeners aren't wired up but we'd still unlisten in the detached.